infrastructure/main.tf

```diff
@@ -96,6 +111,36 @@ resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
   value        = "${module.db.postgresql_database}"
 }
 
+resource "azurerm_key_vault_secret" "new_postgres_user" {
+  key_vault_id = "${module.send-letter-key-vault.key_vault_id}"
+  name         = "${var.component}-new-db-user"
```

Comment:
can we plz name as `${var.component}-db-user-v11` so that it's clear what the secret corresponds to ?

Reply:
sounds good to me. give me a tic